VOC2007/ImageSets/Main/aeroplane_train.txt

```diff
@@ -0,0 +1,2501 @@
+000012 -1
```

Comment:
Alright no problem, I'll remove the files. But merge this branch after I remove the files then.